repo-index.yml

```diff
@@ -512,7 +512,7 @@ plugins:
   version: 1.0.0
   created: 2015-11-11
   updated: 2015-11-11
-  company: 
+  company:
```

Comment:
Clean up and squash commits please. Also, please make whoever you submitted the CLA to aware of this PR and add a comment that they've received it? Cheers.